examples/srd_y10_3x2.yaml

```diff
@@ -0,0 +1,260 @@
+generate:
```

Comment:
Is it true that `srd_y10_3x2.yaml` and `srd_y1_3x2.yaml` only differ in the number of source? Is there a way to avoid too much boilerplate? Ask becker if you don't know.

Reply:
They differ in the number of lens bins too. I'll ask Matt and see if there's anything we can do to avoid so much repetition.